shell.nix

```diff
@@ -0,0 +1,12 @@
+{pkgs ? import <nixpkgs> {}}:
```

Comment:
I can remove this file if you want, it helps having dependencies automatically downloaded when you use nix.

Reply:
@KrzysztofMoch what do you think of this ? I agree to keep it ..

Reply:
It doesn't mind me either so we can leave it